VideoLocker/src/main/java/org/edx/mobile/util/SecurityUtil.java

```diff
@@ -13,20 +12,31 @@
 import java.io.File;
 import java.util.Collections;
 
-public abstract class SecurityUtil {
+/**
+ * Utility class that deals with the security of a user's personal information data.
+ */
+public class SecurityUtil {
     private static final Logger logger = new Logger(SecurityUtil.class);
 
+    private SecurityUtil() {
+    }
+
+    /**
+     * Clears the app's data directory, external storage directory and shared preferences with the
```

Comment:
Never read that **thing**, was never taught that. Its one of those corruption by the people on the other side of the pond that defies logic, physics, chemistry, sense and what not.

Yeah! you read it right, I did'nt use the serial comma again before that **and**.

Let it be too strong. 🤓